classes/Cart.php

```diff
@@ -4323,8 +4323,12 @@ public function deleteCustomizationToProduct($id_product, $index)
             AND `in_cart` = 0'
         );
 
+        if (!$cust_data) {
```

Comment:
Just to be sure and to understand this code: if the previous query fails (then `$cust_data` is empty), do we need to return `true` (the current value for `$result`) or `false` ?

Reply:
If the query returns 0 rows `$cust_data === false`.

Reply:
I confirm that the query is correct.